eman2/convert/convert.py

```diff
@@ -29,16 +29,19 @@
 # **************************************************************************
 
 import glob
+import itertools
 import json
 import numpy
 import os
 
 import pwem.constants as emcts
 import pyworkflow.utils as pwutils
 from pwem.objects.data import Coordinate, Particle, Transform
+from pyworkflow.object import Float
 from pwem.emlib.image import ImageHandler
 import pwem.emlib.metadata as md
 
+from eman2.constants import TOMO_NEEDED_MSG
```

Comment:
as @pconesa noted, this can go

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/3b805869019181e3e364f9837a73ef99f09c12dd